src/makeRobotAccountant.js

```diff
@@ -6,7 +6,19 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+  let counter = 0;
+
+  return function getSum(a) {
+    counter++;
+
+    if (counter <= 3) {
+      return (b) => a + b;
+    } else if (counter % 2 === 0) {
+      return () => 'Bzzz... Error!';
+    } else {
+      return b => a + b;
+    }
```

Comment:
```suggestion
    if (counter % 2 === 0 && counter > 3) {
      return () => 'Bzzz... Error!';
    }
    
    return b => a + b;
```
You can remove extra else...ifs
![image](https://github.com/mate-academy/js_sum-with-closure/assets/96409563/764d2bf3-aa6c-458e-843c-a2f62747f133)